_includes/header.html

```diff
@@ -2,10 +2,14 @@
   <div class="container">
     <a class="logo" href="/">
       <div class="logo__2023">
-        <!-- <img src="{{ site.baseurl }}/img/logo-coworkingfestival-2023.svg" alt="2023" /> -->
+        <img src="{{ site.baseurl }}/img/logo-coworkingfestival-2024.svg" alt="2024" />
```

Comment:
```suggestion
        <img src="{{ site.baseurl }}/img/logo-coworkingfestival-2024.svg" alt="Coworking Festival Logo 2024" />
```

Reply:
It's intended to just say 2024 – that is a better experience for both screenreaders and in cases where the logo is not loading. As the logo says just "2024", the fallback text/screenreader will read exactly that.